### ISSUE TITLE
ReadMe Does not match the latest Docker image release

### ISSUE THREAD

**Issue by jbigel277** [telia-oss/github-pr-resource#206]
The current ReadMe describes the use of an additional parameter (https://github.com/telia-oss/github-pr-resource#get) under "get" where "submodules" can be added however that functionality is not in the v0.21.0 release. The code is already in the master branch a new release just needs to be cut.

**Comment by itsdalmo** [comment]
You can use the `dev` tag to test features currently on master. Unfortunately I think we need to merge #200 or find a better solution (without `--global`) before we can make a new release.

**Comment by jbigel277** [comment]
Okay so I switched to the dev tag but now I have this error
"Host key verification failed."
 We don't use a host key. The reason we've switched from "jtarchie/pr" to "eliaoss/github-pr-resource" is because it's supposed to be using HTTPS and we have a key for that.

**Comment by itsdalmo** [comment]
Yep, I think the cause of the error you are seeing is being worked on in #200. Will have to wait until that lands on master to recursively fetch submodules unfortunately.
